vignettes/hep.Rmd

```diff
@@ -33,6 +34,7 @@ library(billionaiRe)
 hep_df %>%
   transform_hep_data() %>%
   calculate_hep_components() %>%
+  dplyr::bind_rows(billionaiRe::calculate_hep_prevent_ind(.)) %>%
```

Comment:
I'm wondering if this function should pass through the original data frame rather than binding on here to match the other external function set ups